src/SFA.DAS.AssessorService.Domain/Consts/CertificateStatus.cs

```diff
@@ -15,5 +15,6 @@ public class CertificateStatus
         public const string Deleted = "Deleted";
         public const string Reprint = "Reprint";
         public const string Error = "Error";
+        public const string Queued = "Queued";
```

Comment:
This new status needs to be reflected in other parts of the system to ensure Certificates are in the correct status so they can be processed correctly.

I'd imagine most of these will be calls to GetCertificate / GetCertificates and validation handlers (i.e. there's quite a bit in External API for instance)

Reply:
In most cases we are filtering before we are do anything with certificates with the following flags  Domain.Consts.CertificateStatus.Submitted,
               Domain.Consts.CertificateStatus.Printed,
               Domain.Consts.CertificateStatus.Reprint
So in those cases the Queued will be ignored, which is good since this state a transient state and the certificates should not stay in this state unless something went wrong during printing process.
Once the certificate is in submitted state it does not get modified after that until it gets printed.

Reply:
Removed the dependency on Queue